saw-central/src/SAWCentral/Crucible/LLVM/Builtins.hs

```diff
@@ -2769,9 +2770,9 @@ llvm_ghost_value ::
 llvm_ghost_value ghost val = LLVMCrucibleSetupM $
   ghost_value ghost val
 
-llvm_spec_solvers :: SomeLLVM MS.ProvedSpec -> [String]
+llvm_spec_solvers :: SomeLLVM MS.ProvedSpec -> [Text]
 llvm_spec_solvers (SomeLLVM ps) =
-  Set.toList $ solverStatsSolvers $ view MS.psSolverStats $ ps
+  map Text.pack $ Set.toList $ solverStatsSolvers $ view MS.psSolverStats $ ps
```

Comment:
Similarly, I wonder if `solverStatsSolvers` should yield `Set Text` instead of a `Set String` to avoid this intermediate `map Text.pack`.